include/rcutils/logging.h

```diff
@@ -188,23 +188,25 @@ void rcutils_log(
  * The messages with a severity `DEBUG` and `INFO` are written to `stdout`.
  * The messages with a severity `WARN`, `ERROR`, and `FATAL` are written to
  * `stderr`.
- * For each message the severity and name is prepended and the location
+ * The console output format of the logged message can be configured through
+ * the `RCUTILS_CONSOLE_OUTPUT_FORMAT` environment variable.
+ * By default, the severity and name is prepended and the location
```

Comment:
The location seem to be appended in any case (not only when available).

Reply:
thanks, 83a458935bae4e1dc8a4a4b1ae95943237e78f98